README.md

```diff
@@ -42,18 +35,22 @@ Documentation
 Support
 -------
 
-This project is not currently supported due to the presence of
-technically superior alternatives and programming languages.
-Please see https://github.com/aspiers/mysqldiff/issues/14
+Patches should be sent as pull requests to http://github.com/aspiers/mysqldiff
 
-Known bugs
-----------
+Please see the CONTRIBUTING file for more information.
+
+New bug reports and feature requests
+------------------------------------
+
+https://github.com/aspiers/mysqldiff/issues
+
+Other known bugs
+----------------
 
 See https://rt.cpan.org/Public/Dist/Display.html?Name=MySQL-Diff
 
-Copyright
----------
+License
+-------
 
-(c) 2000-2011 Adam Spiers <mysqldiff@adamspiers.org>, all rights reserved.
```

Comment:
Please don't remove copyright signs!  Taking over maintainership doesn't mean that you also take over copyright ;-) Please see https://github.com/aspiers/mysqldiff/commit/572fd93463cf7412f5e564e0c1af21c4cd379070 for a suggested approach; I submitted that branch last night but forgot to push to `master`.